python/llm/test/benchmark/perplexity/ppl_csv_to_html.py

```diff
@@ -37,8 +37,8 @@ def nonzero_min(lst):
     non_zero_lst = [num for num in lst if num > 0.0]
     return min(non_zero_lst) if non_zero_lst else None
 
-def is_diffs_within_normal_range(diff_en, diff_zh, threshold=5.0):
-    return not any(diff < (-threshold) for diff in diff_en + diff_zh if isinstance(diff, float))
+def is_diffs_within_normal_range(diff_ppl_result, threshold=5.0):
+    return not any(diff < (-threshold) for diff in diff_ppl_result + diff_ppl_result if isinstance(diff, float))#diff_en + diff_zh 
```

Comment:
`for diff in diff_ppl_result` not `for diff in diff_ppl_result + diff_ppl_result`